lib/config/pricing/src/index.js

```diff
@@ -148,7 +151,7 @@ const id = (oid, rtype, ppid, time, auth, cb) => {
   if(!rtype) {
     debug('Undefined resource type');
     return cb(undefined,
-      businessError(true, 'Pricing config - Undefined resource type'))
+      businessError('Pricing config - Undefined resource type'))
```

Comment:
Looks like you've created a pattern here, where your error messages have two parts:
1. sometimes the service that detects the error (collector), or the type of entity that causes it (pricing config)
2. a message (undefined resource type) that can only be understood in the context of (1)

I'd suggest to change these error messages as follows:
- do not include the service name as that won't be relevant to the business guy looking at the error
- use a simple descriptive sentence
- include in that sentence enough info for the business guy to know what to do with it

Here's an initial attempt at doing that for the above error: Pricing config for resource instance 'abcd-1234-def-6789' couldn't be determined as its resource type 'foobar-type' is not a known resource type.

An alternative could be return much more structured data with the error, something like:
```
{
  resource_instance_id: 'abcd-1234-def-6789',
  error: true,
  reason: 'pricing config could not be determined'
  cause: {
    resource_type: 'foobar-type',
    error: true,
    reason: 'unknown resource_type'
  }
```

The more structured approach would allow you to describe the sequence of events that caused an error, providing useful info to the business guy who will need to resolve the problem.

Thoughts?